.gitignore

```diff
@@ -89,3 +89,4 @@ dist
 # if you generate sphinx docs, it builds a dummy version of the schema in the idd folder, just ignore it
 /idd/Energy+.schema.epJSON
 /idd/Energy+.schema.epJSON.in
+design/FY2023/earth_tube_solution_space_diagram.pdf
```

Comment:
Why would this file be ignored?  I don't anticipate it changing, so it shouldn't matter much, but I think it should be included.

Reply:
Ok.  I had to go back and look up what was going on here.  It's essentially a duplicate file.  I created the diagram in MS Powerpoint, saved it as a PDF, then opened in Preview and saved it as a .png to match the file type of other documents.  Since it was a duplicate and just a transition file, I didn't think it was worth cluttering the GitHub space.  It's essentially the same as the file of the same name with a .png extension.  You still want this?  I'm ok either way.